docs/src/components/Feedback/index.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import styles from './styles.module.css';
+export const Feedback = () => {
+    return(
+        <div className={styles.wrapper}>
+            <h4>Was this page helpful? We appreciate your feedback</h4>
```

Comment:
can we make use of the translation file here, also in some other places.

Reply:
since these changes will be big, we will do it in separate PR.